test/jdk/javax/swing/JMenuItem/RightLeftOrientation.java

```diff
@@ -22,18 +22,36 @@
  */
 
 /*
- * @test
+ * @test id=metal
  * @bug 4211052
  * @requires (os.family == "windows")
- * @summary
- *     This test checks if menu items lay out correctly when their
+ * @summary Verifies if menu items lay out correctly when their
  *     ComponentOrientation property is set to RIGHT_TO_LEFT.
- *     The tester is asked to compare left-to-right and
- *     right-to-left menus and judge whether they are mirror images of each
- *     other.
  * @library /java/awt/regtesthelpers
  * @build PassFailJFrame
- * @run main/manual RightLeftOrientation
+ * @run main/manual RightLeftOrientation metal
+ */
+
+/*
+ * @test id=motif
+ * @bug 4211052
+ * @requires (os.family == "windows")
+ * @summary Verifies if menu items lay out correctly when their
+ *     ComponentOrientation property is set to RIGHT_TO_LEFT.
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual RightLeftOrientation motif
+ */
```

Comment:
Test can be extended for "Nimbus" L&F also as it is supported for all platforms.

Reply:
Test enhancement can be done in separate PR..even it can be extended for other platforms..but this PR only caters to the problem at hand..

Reply:
In that case, you can raise a JBS issue to extend the test for other platforms.